packages/authgear-react-native/ios/AGWKWebViewController.m

```diff
@@ -23,6 +23,16 @@ - (instancetype)initWithURL:(NSURL *)url redirectURI:(NSURL *)redirectURI comple
     self.completionHandler = completionHandler;
 
     WKWebViewConfiguration *configuration = [[WKWebViewConfiguration alloc] init];
+
+    // Inject `user-select: none` style
+    WKUserContentController *controller = [[WKUserContentController alloc] init];
+    NSString *disableUserSelectScript =
+        @"document.documentElement.style.webkitUserSelect = 'none';"
+        "document.documentElement.style.userSelect = 'none';";
+    WKUserScript *userScript = [[WKUserScript alloc] initWithSource:disableUserSelectScript injectionTime:WKUserScriptInjectionTimeAtDocumentEnd forMainFrameOnly:NO];
+    [controller addUserScript:userScript];
+    configuration.userContentController = controller;
```

Comment:
I suppose configuration.userContentController is nil, so you need to create a WKUserContentController yourselves? If this is the case, you should leave a comment to document this behavior difference between Objective-C and Swift.

Reply:
I have tested, and it is absolutely fine to just use configuration.userContentController. I added a commit to correct this.